x-pack/plugins/uptime/scripts/e2e.js

```diff
@@ -59,17 +59,25 @@ if (server) {
 const config = './playwright_run.ts';
 
 function executeRunner() {
-  if (runner) {
+  if (server) {
     childProcess.execSync(
-      `node ../../../scripts/${ftrScript} --config ${config} --kibana-install-dir '${kibanaInstallDir}' --headless ${headless} --grep ${grep}`,
+      `node ../../../scripts/${ftrScript} --config ${config} --kibana-install-dir '${kibanaInstallDir}'`,
+      {
+        cwd: e2eDir,
+        stdio: 'inherit',
+      }
+    );
+  } else if (runner) {
+    childProcess.execSync(
+      `node ../../../scripts/${ftrScript} --config ${config} --kibana-install-dir '${kibanaInstallDir}'  --headless ${headless} --grep '${grep}'`,
```

Comment:
not sure, why this got changed?

Reply:
The original else is the condition that is executed in CI, as it is neither runner nor server, and runs `functional_tests.js`. For that, I needed to add `--grep`. But when you're using runner, you also want `--headless`, which `functional_tests.js` cannot accept. So each of these, server, runner, and functional_tests are all different in what args they need.